components/iotserver-mgt/org.wso2.carbon.device.mgt.iot.common/src/main/java/org/wso2/carbon/device/mgt/iot/common/transport/mqtt/MqttConfig.java

```diff
@@ -0,0 +1,55 @@
+package org.wso2.carbon.device.mgt.iot.common.transport.mqtt;
+
+import org.wso2.carbon.device.mgt.iot.common.config.server.DeviceCloudConfigManager;
+import org.wso2.carbon.device.mgt.iot.common.config.server.datasource.ControlQueue;
+
+public class MqttConfig {
+	private String mqttQueueEndpoint;
```

Comment:
Add a newline between the class signature and the initialization of mqttQueueEndpoint variable.